examples/ar2.py

```diff
@@ -64,8 +64,6 @@ def transition(carry, _):
     with numpyro.handlers.condition(data={"y": y[2:]}):
         _, mu = scan(transition, init, timesteps)
 
-    numpyro.deterministic("mu", mu)
```

Comment:
Is this intended?

Reply:
No 🤦‍♂️

Reply:
Fixed in https://github.com/pyro-ppl/numpyro/pull/1816/commits/8ebbaa7bf4a072c2fd4e60ef0eaef3abfad6795e